servers/cromwell/README.md

```diff
@@ -53,7 +54,8 @@ Thin shim around [`cromwell`](https://github.com/broadinstitute/cromwell).
      "field": "labels.label",
      "display": "Label",
      "fieldType": "text",
-     "editable": true
+     "editable": true,
+     "bulkEditable": true
```

Comment:
The last item in `displayFields` here is missing a closing `}`

Reply:
thanks catching that – fixed